js/views/search/Search.js

```diff
@@ -27,6 +28,21 @@ export default class extends baseVw {
 
     super(opts);
     this.options = opts;
+
+    this.defaultSuggestions = this.options.defaultSuggestions ||
+      [
+        'books',
```

Comment:
hopefully @tyler-smith could send back some more useful suggestions, because none of these result in any results.

I recommend this wonderful restaurant for dinner tonight -- the restaurant is only open for breakfast and lunch :(

Reply:
nevermind t-sizzle. I was on testnet... lol!